### PR TITLE
Add details page

### PR DESCRIPTION
**Pull request by DelhinRharl**
## Hi there!
In this pull request,
 

- [ ] I added the details page

### CODE REVIEW

File: src/pages/CarDetails.js [hunk 1]
```diff
@@ -1,3 +1,36 @@
-const CarDetails = () => <h1>Car details page</h1>;
+import React from 'react';
+import carImageBig from '../images/car-big.png';
+
+const CarDetails = () => (
+  <div className="flex flex-col gap-[30%] p-2 items-center justify-center">
```
Comment (anuarshaidenov) [inline src/pages/CarDetails.js]: For the main div element, kindly add `md:w-4/5`, so that the sidebar is more consistent

**Review by anuarshaidenov — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED♻️
Well done with making the page responsive, @DelhinRharl, however, I suggest adding width to the main container so that the sidebar is more consistent. Kindly check the comments under

File: src/pages/CarDetails.js (hunk 1 above)
Comment (DelhinRharl) [reply src/pages/CarDetails.js]: Alright...thank you

**Review by anuarshaidenov — APPROVED** [review]
## STATUS: APPROVED🎉
@DelhinRharl **Well done! Your feature is complete, you can now merge your branch🙂**

**Pull request merged**